cypress/integration/save.cy.js

```diff
@@ -90,6 +94,59 @@ describe('save', () => {
             .its('response.body')
             .should('have.property', 'sorting')
 
+        expectTableToBeVisible()
+
+        getTableRows()
+            .eq(0)
+            .find('td')
+            .eq(1)
+            .invoke('text')
+            .then(parseInt)
+            .then(($cell0Value) =>
+                getTableRows()
+                    .eq(1)
+                    .find('td')
+                    .eq(1)
+                    .invoke('text')
+                    .then(parseInt)
+                    .then(($cell1Value) =>
+                        expect($cell0Value).to.be.lessThan($cell1Value)
```

Comment:
maybe technically correct, but why not just check for specific values? like expect cell0 to be 15 and cell1 to be 20 etc. That's usually the preferred reliable way.

Reply:
Refactored.